consumer/consumererror/permanenterror.go

```diff
@@ -15,7 +15,7 @@
 // Package consumererror provides wrappers to easily classify errors. This allows
 // appropriate action by error handlers without the need to know each individual
 // error type/instance.
-package consumererror
+package consumererror // import "go.opentelemetry.io/collector/consumer/consumererror"
```

Comment:
We should do this for the entire repo I think, so probably not necessary right now?